src/Agent.Plugins/PipelineCache/PipelineCacheServer.cs

```diff
@@ -193,10 +188,10 @@ private async Task<string> GetUploadPathAsync(string preferredContentFormat, Age
             DedupManifestArtifactClient dedupManifestClient,
             DedupIdentifier manifestId,
             string targetDirectory,
-            string contentFormat,
+            ContentFormat contentFormat,
             CancellationToken cancellationToken)
         {
-            if (contentFormat == ContentFormatConstants.SingleTar)
+            if (contentFormat == ContentFormat.SingleTar)
```

Comment:
I hope I am not out of place here - i was just looking around hehe I think this could be further improved to be future-proof by respecting the Open Close Principle and delegating this to another object instead of stacking if/elseif/elseif/else in the future